Resources/Prototypes/Entities/Constructible/Items/metal_rod.yml

```diff
@@ -0,0 +1,25 @@
+﻿- type: entity
+  name: Metal Rod
+  parent: BaseItem
+  id: MetalRod
```

Comment:
Make this has a "full" suffix after you address my review below.
Also make another prototype that inherits this one that only has 1 metal rod in the stack and give that the "1" suffix or something. Check how other sheets/materials did it for consistency.